### DISCUSSION TITLE
[Q-Table] Dynamically updating the number of rows in QTable

### DISCUSSION THREAD

**Discussion by JekaKey** [quasarframework/quasar#16635] [General - Components / Directives / etc] [answered]
Hello Quasar Community!
I can't update the number of rows in a table when dynamically loading into a table using the  Options API.
The table always uses the initial value. What is the correct way to update this value?
Simple example in [CodePen](https://codepen.io/JekaKey/pen/KKJBMbg)

**Comment by pdanpdan** [comment, marked as answer]
Use `v-model:pagination="pagination"` as specified in the API card in the docs

**Comment by JekaKey** [comment]
Thank you Sir!
